utils/cliutils/commandsflags.go

```diff
@@ -1259,6 +1260,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  ExtendedTable,
 		Usage: "[Default: false] Set to true if you'd like the table to include extended fields such as 'CVSS' & 'Xray Issue Id'. Ignored if provided 'format' is not 'table'. ` `",
 	},
+	AuditUseWrapper: cli.BoolTFlag{
+		Name:  AuditUseWrapper,
+		Usage: "[Default: True] Set to fale if you wish to not use the wrapper ",
```

Comment:
True --> true
...wish to not use the wrapper --> ...wish to not use the Gradle or Maven Wrapper.